### ISSUE TITLE
[2.4.2-beta-2] Fix TreeView rendering when using  globalizeObjectIds

### ISSUE THREAD

**Issue by mlankamp** [xeokit/xeokit-sdk#1224]
**Describe the bug**
When using the option to globalize ObjectIds the treeview doesn't render

**To Reproduce**
Use one of the xkt examples, and add the following line:
```diff
const xktLoader = new XKTLoaderPlugin(viewer);
+ xktLoader.globalizeObjectIds = true;
```

The same behavior happens when using:
```diff
    const sceneModel = xktLoader.load({
        id: "myModel",
        src: "../../assets/models/xkt/v8/ifc/IfcOpenHouse4.ifc.xkt",
        saoEnabled: true,
        edges: true,
+        globalizeObjectIds: true,
        objectDefaults: { // This model has opaque windows / spaces; make them transparent
            "IfcPlate": {
                opacity: 0.3 // These are used as windows in this model - make transparent
            }
        }
    });
```

**Expected behavior**
The treeview should render.

Or is this setting depricated?

**Comment by xeolabs** [comment]
Yip that's a bug. We'll fix as priority.

`globalizeObjectIds` is not deprecated, but the only use case we have that AFAIK is when we want to load two or more instances of the same model alongside each other. Do you happen to know of other user cases?

**Comment by mlankamp** [comment]
No, I don't know any other use case.

**Comment by xeolabs** [comment]
Fixed in 2.4.2-beta-3
